test/vanilla/ignite-new.test.ts

```diff
@@ -28,8 +17,16 @@ describe("Checking for ignite. 🪔", () => {
 })
 
 describe("Igniting new app! 🔥\nGo get a coffee or something. This is gonna take a while.", () => {
+  const originalDir = process.cwd()
+  const tempDir = path.join(originalDir, "temp")
+  const appDir = path.join(tempDir, APP_NAME)
+
+  beforeEach(() => {
+    filesystem.remove(appDir)
+  })
```

Comment:
`tempy` was having some strange behavior on my windows machine, sometimes putting the file into inaccessible file locations and causing later tests to fail after not being able to `cd` back and forth from `tempDir` to `originalDir`. This way, it is always deterministic. 

I also changed the test to only clean the directory before each tests so I could inspect the `temp/Foo` output on failing tests.

Might be worth moving this into the `test/` directory to make it more clear of it's purpose.

Reply:
Is this a known issue with tempy on Windows? I am reluctant to change this for erratic Windows behavior unless we have to.

Reply:
Upon some further testing, the test suite fails on windows for a variety of reasons. 

`tempy` may be an issue on windows, but I've switched to WS.

See more here: L https://github.com/infinitered/ignite/pull/2009#discussion_r946208419